### PR TITLE
Bump Ruby versions for Travis

### PR DESCRIPTION
**Pull request by januszm**
* latest 2.2 and 2.3
* 2.4.0

### CODE REVIEW

**Comment by januszm** [comment]
cc @swrobel it might require more work, there are some issues with Rails 4.x

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,9 @@ before_install:
   - gem install bundler
 rvm:
   - ruby-head
-  - 2.3.3
-  - 2.2.6
+  - 2.4.0
```
Comment (swrobel) [inline .travis.yml]: Why not 2.4 and let it automatically use the latest 2.4.x instead of locking to 2.4.0?

Comment (januszm) [inline .travis.yml]: because RVM won't work with it and Travis uses RVM

Comment (swrobel) [inline .travis.yml]: uuuuggggggh, classic RVM move!

**Comment by januszm** [comment]
what can you do :)

**Comment by januszm** [comment]
They'll fix it around 2.4.1 I guess and we'll be able to switch to '2.4'

**Pull request merged**